Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -14,8 +14,10 @@
 from Configuration.Eras.Modifier_run2_HLTconditions_2016_cff import run2_HLTconditions_2016
 from Configuration.Eras.Modifier_run2_muon_2017_cff import run2_muon_2017
 from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
+from Configuration.Eras.Modifier_run2_egamma_2017_cff import run2_egamma_2017
+from Configuration.Eras.Modifier_run2_egamma_2016_cff import run2_egamma_2016
 
-Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016]), 
+Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016,run2_egamma_2017]), 
```

Comment:
this doesn't look right: it should be ```run2_egamma_2016``` here (of the things to be excluded from ```Run2_2016``` when making ```Run2_2017```

Reply:
done, thanks for the catch